Add pages related to Sign-up mode Restricted (#1562)
clerk/clerk-docs: docs/authentication/configuration/sign-up-sign-in-options.mdx

1. Select the user from the list.
1. Select the **Delete MFA enrollments** button.

## Disable sign-ups

> Clerk offers different sign-up modes to help you control how new users access your application. Switching to the "Restricted" mode disables new user sign-ups, ensuring that only pre-approved users can access your application. For more details, see our [sign-up modes page](/docs/authentication/configuration/sign-up-modes).

Reviewer: ```suggestion
Clerk offers different sign-up modes to help you control how new users access your application. Switching to the **Restricted** mode disables new user sign-ups, ensuring that only pre-approved users can access your application. For more information, see [the dedicated guide](/docs/authentication/configuration/sign-up-modes).
```

## Restrictions

> Clerk provides a set of restriction options designed to provide you with enhanced control over who can gain access to your application. Restrictions can limit sign-ups or prevent accounts with specific identifiers from accessing your application. These identifiers can be email addresses, phone numbers, and even entire domains. Learn more about restrictions in our [detailed guide](/docs/authentication/configuration/restrictions).

Reviewer: ```suggestion
Clerk provides a set of restriction options designed to provide you with enhanced control over who can gain access to your application. Restrictions can limit sign-ups or prevent accounts with specific identifiers from accessing your application. These identifiers can be email addresses, phone numbers, and even entire domains. Learn more about restrictions in [the dedicated guide](/docs/authentication/configuration/restrictions).
```
